Vonk.Plugin.DocumentOperation.Test/TestResourceReader.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;

```

Comment:
You don't need this whole class. The API simply uses File.ReadAllText - that is probably more understandable in a public plugin.